packages/angular_devkit/build_angular/src/babel/webpack-loader.ts

```diff
@@ -107,18 +115,35 @@ export default custom<ApplicationPresetOptions>(() => {
 
       // Analyze for ES target processing
       const esTarget = scriptTarget as ScriptTarget | undefined;
-      if (esTarget !== undefined) {
-        if (esTarget < ScriptTarget.ES2015) {
-          customOptions.forceES5 = true;
-        } else if (esTarget >= ScriptTarget.ES2017 || /\.[cm]?js$/.test(this.resourcePath)) {
+
+      // The below should be dropped when we no longer support ES5 TypeScript output.
+      if (esTarget === ScriptTarget.ES5) {
```

Comment:
This is to retain the same behaviour, but eventually we should start ignoring the TypeScript target and rely on the browserslist configuration. Although that does seem like a breaking behaviour therefore probably that must happen in v15?

Reply:
To make sure I understand, is the behavior here that when setting `target: "es5"`, we effectively ignore browserslist and always downlevel everything to ES5? Does that change existing application code, or only libraries because they are now downeleveled too?

I had a quick conversation with the rest of the leads and several parts of Angular no longer work on IE (some components and some security / sanitization stuff), but there are still aspects which could work and I think we're unwilling to actively break such use cases just yet.

We can probably emit a warning in this case (whether you want to do that in this PR or another one) to be clear that this isn't supported and the user is probably doing something they shouldn't be.

Reply:
This maintains the existing behavior of downlevelling to ES5 compatible code for *both* application code and libraries when the TS target is set to `es5`.  The browserslist will continue to be ignored in this case.

Reply:
Ah right, we were already doing that for ES5. So now we're doing it for all language levels, it's just that normally it's dependent on browserslist, but setting `target: "es5"` overrides that. I agree we should get rid of that override eventually, but I'm fine waiting for v15 to make that happen.